### PR TITLE
Propagate captures from `before` section

### PR DESCRIPTION
**Pull request by RemiBardon**
While documenting the `before` and `after` sections (#118) I realized I didn't make it so captures from `before` are available in `tests` and `after`. This PR fixes it.

### CODE REVIEW

**Comment by mishushakov** [comment]
Alrighty, have you tested it?

**Comment by RemiBardon** [comment]
I wouldn't have submitted otherwise 😌

File: src/index.ts [hunk 1]
```diff
@@ -253,18 +253,22 @@ export async function run(workflow: Workflow, options?: WorkflowOptions): Promis
   const limit = pLimit(concurrency <= 0 ? 1 : concurrency)
 
   let testResults: TestResult[] = []
```
Comment (mishushakov) [inline src/index.ts:255]: change it to `const`?

Comment (RemiBardon) [reply src/index.ts]: Done

**Comment by RemiBardon** [comment]
By the way, you can see `{ ...captures }` when running `tests`. This is because

> Any variable captured in the `before` section will be available in the `tests` and `after` sections. However, since variables captured in individual tests are not shared to other tests, any variable captured in the `tests` section will **not** be available in the `after` section.

(From https://github.com/stepci/stepci/pull/225)

**Comment by mishushakov** [comment]
Okay for me

**Pull request merged**